docs/troubleshooting.rst

```diff
@@ -349,8 +349,7 @@ If you want to know if your problem is indeed with the dynamic patcher, you can
           yield
 
 
-  def test_something(fs_no_dyn_patch):
-      ...  # do the testing
+  def test_something(fs_no_dyn_patch): ...  # do the testing
```

Comment:
I'm not too happy with these, as the ellipses are meant as a replacement for code, and not as an actual ellipsis as assumed by the check. Probably replace them with some dummy function, e.g. something like:
`assert foo()  # do the testing`?
Same for the other changes regarding the ellipsis.

Reply:
I think this is addressed now.